wundergraph/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@sidestream/wundergraph",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "wunderctl up",
+    "dev": "run-p dev:wundergraph open",
+    "dev:wundergraph": "wunderctl up --logs",
+    "build:wundergraph": "wunderctl generate --debug",
+    "open": "wait-on -d 500 http://localhost:9991 && open-cli http://localhost:3000 && open-cli http://localhost:3001 && open-cli http://localhost:4000"
```

Comment:
What is the `http://localhost:9991` in this case?

Reply:
WG graphql endpoint